samplers/jaegerremote/version.go

```diff
@@ -16,7 +16,7 @@ package jaegerremote // import "go.opentelemetry.io/contrib/samplers/jaegerremot
 
 // Version is the current release version of the Jaeger remote sampler.
 func Version() string {
-	return "0.30.0"
+	return "0.2.0"
```

Comment:
The other two samplers seem to be missing `version.go`, should we add them here?

Reply:
Will do :+1: